release.config.js

```diff
@@ -0,0 +1,11 @@
+// https://semantic-release.gitbook.io/semantic-release/usage/configuration
+module.exports = {
+  branches: [
+    '+([0-9])?(.{+([0-9]),x}).x',
+    'main',
```

Comment:
Replaces the default `master` with `main`.

Reply:
This doesn't seem to get picked up when running `npx semantic-release` locally, but from what I can tell, it needs to pushed up to the remote for it to work locally, too.